CONTRIBUTING.md

```diff
@@ -43,6 +43,13 @@ fitting the following criteria:
  * Lastly, the functionality conforms to the contribution guidelines of
  its subpackage.
 
+Suggested guidelines for new feature requests:
+
+* The feature contains an official refrence implementation.
+* Should be able to reproduce the same results in a published paper.
+* Should provide out-of-box performance.
```

Comment:
Not very clear to the end user. What does it mean?

Reply:
I guess what I'm trying to articulate here is that the feature is recommended to contain a relevant DL academic paper. In addition, the feature being implemented should be able to perform with similar results from that of the relevant DL academic paper.

Reply:
I was pointing out this point specifically `Should provide out-of-box performance.`

Reply:
Great! Thanks for the suggestions. I apologize for my untimely responses since I've been busy studying for my midterms 😢.

Reply:
No worries. Thanks for the contribution